clusters.py

```diff
@@ -16,21 +16,18 @@
 import datetime
 
 import gspread
-import google.auth as auth # authenticatiing to google
+import google.auth as auth # authenticating to google
 
 #Global variables/parameters
 
-default_cluster_dir = ('/content/drive/MyDrive/Stanford Audiology Models/'
-                       'Colab Notebooks/')
+default_cluster_dir = ('/content/drive/MyDrive/STAR_Audiology_Model/')
```

Comment:
We should probably talk and agree on a default that makes sense for both of us.  Not sure what that might be... perhaps the current directory (and then we can put a link into the place we each use.)?